simplq/src/components/common/Button.stories.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import StandardButton from './Button';
+
+export default {
+  component: StandardButton,
+  title: 'Button',
+};
+
+export const CreateQButton = (props) => {
+  return <StandardButton onClick={props.onClick} text="Create a Queue" />;
+};
+
+export const JoinQButton = (props) => {
+  return <StandardButton onClick={props.onClick} text="Join a Queue" />;
+};
```

Comment:
So this will be the general pattern we will follow throughout?

An extra thing, how about we move the button component and it's story to a `Button` folder, then have a index.js in the Button folder which just exports all the buttons...? Then anyone wanting to use them can import them from `src/components/common/Button`.

Otherwise at the component side, imports from the `.stories` file will come, and won't look good. We want to encapsulate storybook here.